migrations/20211118103939_add_deleted_user_record.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = function (knex) {
+  return knex.raw(`
+    INSERT INTO users(email, first_name, last_name, active, email_validated, bio)
+    VALUES ('deleted@hylo.com', 'Deleted', 'User', false, true, 'This is the generic "deleted user" account');
```

Comment:
should we create this email account for some reason maybe?

Reply:
I was leaning towards no, but is there a compelling reason to do so? Will some functionality break if we don't have a real email associated with this account?

Reply:
i cant think of anything that would break right now. i wonder if/how we can turn off notifications for this user? also do we need to add this user to the groups that posts are in? we will need to test the crap out of this all